uavcan/tunnel/2011.SerialConfig.uavcan

```diff
@@ -0,0 +1,9 @@
+#
+# This message carries baudrate for tunnel channel_id
+
+# The channelID allows for additional routing between the source and target nodes.
+uint8 channel_id
+
+uint32 baud
+
+uint32 options
```

Comment:
How are the options encoded? Or would this be additional enums?

Reply:
additional enums will be required, I am letting them not defined until there is actual implementation.